examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
```

Comment:
That is certainly much too strict. Part of the reason why I don't like ESLint is because it seems that most people who use it just do so to make sure they're on the "latest greatest" without any regard for what that actually *means*. For example, arrow functions don't obsolete anonymous functions. You still want access to `this` in the context of, say, an EventEmitter callback or (most certainly) in an object method.